symfony/src/Controller/CronController.php

```diff
@@ -20,7 +20,7 @@
 use Symfony\Component\Routing\Annotation\Route;
 
 /**
- * @Route("/s/cron")
+ * @Route("/s/cron", condition="request.headers.get('X-Appengine-Cron') matches '/true/'")
```

Comment:
What's the reason for a regex match here? Do we not want the exact string 'true'?

Reply:
Example code only used `matches`, so I'd assumed that it had to be regular expression. Further reading indicates I can get away with an exact string. (`==` or `===`)

Reply:
Resolved in 6c125e4.